meson.build

```diff
@@ -13,6 +13,7 @@ add_project_arguments(cc.get_supported_arguments([
 	'-Wno-missing-braces',
 	'-Wno-missing-field-initializers',
 	'-Wno-unused-parameter',
+	'-Werror=maybe-uninitialized',
```

Comment:
This is already included in `-Wall`.

Reply:
https://github.com/emersion/xdg-desktop-portal-wlr/pull/59#discussion_r578460525

This was the reason for the commit. I let it separate so it can be ommited on merge if it is not needed. Couldn't reproduce the issue on my system.

Reply:
Some warnings may only trigger at higher optimization levels. Maybe try `debugoptimized`?

In any case, I don't think it makes sense to add this flag.

Reply:
ok try that and remove it, if i can't detect a difference